browser.js

```diff
@@ -5,12 +5,18 @@ module.exports = function () {
     set: function (name, value, options) {
       return document.cookie = cookie.serialize(name, value, options);
     },
+
     get: function (name) {
       return cookie.parse(document.cookie)[name];
     },
-    remove: function (name) {
-      return !!(document.cookie = cookie.serialize(name, '', { expires: new Date(0) }));
+
+    remove: function (name, options) {
+      var opts = options || {};
+      opts.expires = new Date(0);
+
+      return !!(document.cookie = cookie.serialize(name, '', opts));
```

Comment:
The return is the same as what jQuery does, it's basically validating that the document.cookie is set.  I don't believe this is a way to hook into it more to validate the cookie is set correctly w/o doing a bunch of checks.  For now, I think that's a bit out of scope for this PR.